packages/api/tests/models/test-providers-model.js

```diff
@@ -162,6 +162,19 @@ test('Providers.create() allows creation of a provider without a globalConnectio
   t.is(fetchedProvider.globalConnectionLimit, undefined);
 });
 
+test('Providers.create() allows creation of a provider with allowedRedirects', async (t) => {
+  const { providersModel } = t.context;
+
+  const provider = fakeProviderFactory();
+  provider.allowedRedirects = ['foo.com'];
```

Comment:
Is this the format we're expecting? Just the domain name? Mainly asking because of the bug I just fixed where we're expecting a string but it's unclear what that string represents. In that case ARN vs. full URL.

Reply:
Yeah that's a very good question. In this case we are expecting just the domain name, or the domain name with a port (`foo.com:80`) if we're being redirected to a different port than the initial request